public/components/content-list-item/templates/links.html

```diff
@@ -1,27 +1,37 @@
-<td class="content-list-item__field--composer">
+<td class="content-list-item__field--composer" ng-if="contentItem.contentType !== 'media'">
     <a href="{{ contentItem.links.composer }}" title="View in Composer (opens in new window)" target="_blank" ng-if="contentItem.links.composer" ng-click="$event.stopPropagation()">
         <i class="content-list-item__icon--composer" wf-icon="composer"></i>
     </a>
                 <span ng-if="!contentItem.links.composer" title="View in Composer (unavailable)" class="content-list-item__icon--inactive">
                     <i class="content-list-item__icon--composer" wf-icon="composer" wf-icon-active="false"></i>
                 </span>
 </td>
-<td class="content-list-item__field--preview">
+<td class="content-list-item__field--preview"  ng-if="contentItem.contentType !== 'media'">
     <a href="{{ contentItem.links.preview }}" title="Preview (opens in new window)" target="_blank" ng-if="contentItem.links.preview" ng-click="$event.stopPropagation()">
         <i class="content-list-item__icon--preview" wf-icon="preview"></i>
     </a>
                 <span ng-if="!contentItem.links.preview" title="Preview (unavailable)" class="content-list-item__icon--inactive">
                     <i class="content-list-item__icon--preview" wf-icon="preview" wf-icon-active="false"></i>
                 </span>
 </td>
-<td class="content-list-item__field--live">
+<td class="content-list-item__field--live"  ng-if="contentItem.contentType !== 'media'">
```

Comment:
This can probably be done later, but is this check  `contentItem.contentType !== 'media'` going to be `!== 'atomofanykind` on these at some point?